addons/mail/__manifest__.py

```diff
@@ -135,9 +135,7 @@
             # depends on BS variables, can't be loaded in assets_primary or assets_secondary
             'mail/static/src/scss/variables/derived_variables.scss',
             'mail/static/src/scss/*.scss',
-            'mail/static/lib/**/*',
-            ('remove', 'mail/static/lib/odoo_sfu/odoo_sfu.js'),
-            ('remove', 'mail/static/lib/lame/lame.js'),
+            'mail/static/lib/selfie_segmentation/selfie_segmentation.js',
```

Comment:
Shouldn't selfie_segmentation be lazy-loaded too while we are at it?

Reply:
Maybe, this is just a cleanup because the code doesn't feel right, but I know nothing about the lib.